packages/typespec-test/test/openai_modular/generated/typespec-ts/src/rest/index.ts

```diff
@@ -11,5 +11,11 @@ export * from "./isUnexpected.js";
 export * from "./models.js";
 export * from "./outputModels.js";
 export * from "./pollingHelper.js";
+export {
+  createFile,
+  createFileFromStream,
+  type CreateFileOptions,
```

Comment:
why we need to add 'type' here?

Reply:
createFile would have the its option definition which is CreateFileOptions.